Add import data tutorials to the main tutorials section (#1933)
pyansys/pydpf-core: doc/source/user_guide/tutorials/import_data/represent_data_on_dpf.rst

> .. _ref_tutorials_represent_data_on_dpf:

Reviewer: ```suggestion
.. _ref_tutorials_custom_data:
```

========================
Manual input data on DPF
========================

.. |Field| replace:: :class:`Field<ansys.dpf.core.field.Field>`
.. |FieldsContainer| replace:: :class:`FieldsContainer<ansys.dpf.core.fields_container.FieldsContainer>`
.. |append| replace:: :func:`append()<ansys.dpf.core.field.Field.append>`
.. |data| replace:: :attr:`Field.data<ansys.dpf.core.field_base._FieldBase.data>`
.. |scoping| replace:: :attr:`Field.scoping<ansys.dpf.core.field_base._FieldBase.scoping>`

> This tutorial shows how to represent your manual input data in a DPF data storage structures.

Reviewer: ```suggestion
This tutorial shows how to represent your custom data in DPF data storage structures.
```

> When handling data DPF uses |FieldsContainer| and |Field| to store and return it. The |Field| is a DPF array
> and a collection of |Field| is called |FieldsContainer|. For more information on how the data is structure
> in a |Field| and how the DPF data storage structures works check the :ref:`ref_tutorials_data_structures`
> tutorial section.
> 
> Here we will create some 3d vector |Field|, where the data comes from lists.

Reviewer: ```suggestion
DPF uses |Field| and |FieldsContainer| objects to handle data. The |Field| is a homogeneous array
and a |FieldsContainer| is a labeled collection of |Field|. 
For more information on DPF data structures such as the |Field| and their use check the :ref:`ref_tutorials_data_structures` tutorials section.

Here we show how to create a 3d vector |Field| from data stored in Python lists.
```
We need a part on how to do that with `numpy` arrays.

> Defining the fields
> -------------------

Reviewer: ```suggestion
Create DPF fields
-------------------
```

Reviewer: @luisaFelixSalles I would personally use three sections for this tutorial:
- a first section where we just define the custom data used later
- a second section on how to create a field -> we need to show more here on how to set the proper nature, size and dimensionality. This is crucial to properly set the data later so even if it is already in another tutorial I would still explain the basics here. Also show the field_factory, the fact you can create unitary fields, fields with a given value everywhere, etc. With have helpers so should mention them here.
- a third section where we show how to set data arrays directly, and how to set the scoping for that array
This is the most important as people rarely only add three data points. There are steps missing in this third section however on setting the data property. The scoping is undefined in this case, so it needs to also be built and set.
- a fourth section where we show how to append data or change data for a specific entity id
- a final section on how to create the fields_container based on the fields as you showed with the helper. Mention the fields_container factory etc, without going into too much detail if there is another tutorial for that. To me this is a last step as not everyone will need to create a collection for the fields.

> To manually import data on DPF you have to create the structure to store it.

Reviewer: ```suggestion
To import custom data into DPF you first have to create the structure to store it.
```
